src/Node/GethNode.ts

```diff
@@ -22,34 +22,11 @@ export default class GethNode extends Node {
    * Starts the container that runs this chain node.
    */
   public start(): void {
+    let args = super.setup();
```

Comment:
I moved it because to it would be more readable to create a network before `init`. 
Ofcourse, logging must be the first line.
Let me know what do you think ?

Reply:
Please disregard the in-line comments and revert the changes according to the review comment above. Sorry, I misjudged the required effort/changes of moving the `args` to the parent class.